FlorenceBML/BMLacq681/BMLacq681.xml

```diff
@@ -35,9 +36,139 @@ schematypens="http://relaxng.org/ns/structure/1.0"
+                                            <dim unit="leaf">10</dim>
+                                            <locus from="43r" to="52v"/>
+                                        </item> 
+                                        <item xml:id="q8" n="8">
+                                            <dim unit="leaf">10</dim>
+                                            <locus from="53r" to="62v"/>
+                                        </item> 
+                                        <item xml:id="q9" n="9">
+                                            <dim unit="leaf">8</dim>
+                                            <locus from="63r" to="70v"/>
+                                        </item> 
+                                        <item xml:id="q10" n="10">
+                                            <dim unit="leaf">8</dim>
+                                            <locus from="71r" to="78v"/>
+                                        </item> 
+                                        <item xml:id="q11" n="11">
+                                            <dim unit="leaf">8</dim>
+                                            <locus from="79r" to="86v"/>
+                                        </item> 
+                                        <item xml:id="q12" n="12">
+                                            <dim unit="leaf">9</dim>
+                                            <locus from="87r" to="95v"/>
+                                            s.l. 6, no stub
+                                        </item> 
+                                        <item xml:id="q13" n="13">
+                                            <dim unit="leaf">8</dim>
+                                            <locus from="96r" to="103v"/>
+                                        </item> 
+                                        <item xml:id="q14" n="14">
+                                            <dim unit="leaf">8</dim>
+                                            <locus from="104r" to="111v"/>
+                                        </item> 
+                                        <item xml:id="q15" n="15">
+                                            <dim unit="leaf">2</dim>
+                                            <locus from="112r" to="113v"/>
+                                        </item> 
+                                    </list>
+                                    <item xml:id="q16" n="16">
+                                        <dim unit="leaf">6</dim>
+                                        <locus from="114r" to="119v"/>
+                                        s.l. 4, stub after 5,
+                                        s.l. 5, stub after 5,
+                                        s.l. 7, no stub,
+                                        s.l. 8, no stub
+                                        <note>Contrary to Marrassini, who considered <locus target="#118"/> and <locus target="#119"/> separate from 
```

Comment:
Perhaps, to be more transparent here and above, you can equate folia with the numbers of the leaves in quires 
(who considered <locus target="#118"/> (leave ...) and <locus target="#119"/> (leave ...) separate from...

Reply:
Leaves and folia in brackets